We need tiny, early-userspace aware binares statically linked with dietlibc by
default. Thus no .so.

  - Rene Rebe <rene@exactcode.de>

--- udev-101/extras/volume_id/Makefile.orig	2006-09-30 14:30:00.000000000 +0200
+++ udev-101/extras/volume_id/Makefile	2006-10-11 20:14:42.000000000 +0200
@@ -36,11 +36,7 @@
 
 $(PROG): %: $(HEADERS) %.o lib/libvolume_id.a
 	$(E) "  LD      " $@
-ifeq ($(strip $(VOLUME_ID_STATIC)),true)
 	$(Q) $(LD) $(LDFLAGS) -o $@ $@.o $(LIBUDEV) lib/libvolume_id.a $(LIB_OBJS)
-else
-	$(Q) $(LD) $(LDFLAGS) -o $@ $@.o $(LIBUDEV) -Llib -lvolume_id $(LIB_OBJS)
-endif
 
 # man pages
 %.8: %.xml
--- ./extras/volume_id/lib/Makefile.orig	2006-10-11 20:25:15.000000000 +0200
+++ ./extras/volume_id/lib/Makefile	2006-10-11 20:25:48.000000000 +0200
@@ -58,7 +58,7 @@
 	libvolume_id.h		\
 	util.h
 
-all: libvolume_id.a $(SHLIB) libvolume_id.pc
+all: libvolume_id.a libvolume_id.pc
 .PHONY: all
 .DEFAULT: all
 
@@ -103,9 +103,6 @@
 install: all
 	$(INSTALL_DATA) -D libvolume_id.h $(DESTDIR)$(includedir)/libvolume_id.h
 	$(INSTALL_LIB) -D libvolume_id.a $(DESTDIR)$(usrlibdir)/libvolume_id.a
-	$(INSTALL_LIB) -D $(SHLIB) $(DESTDIR)$(libdir)/$(SHLIB)
-	ln -sf $(SHLIB) $(DESTDIR)$(libdir)/libvolume_id.so.$(SHLIB_CUR)
-	ln -sf $(libdir)/$(SHLIB) $(DESTDIR)$(usrlibdir)/libvolume_id.so
 	$(INSTALL_DATA) -D libvolume_id.pc $(DESTDIR)$(usrlibdir)/pkgconfig/libvolume_id.pc
 .PHONY: install
 
